qcengine/programs/qchem.py

```diff
@@ -59,29 +61,45 @@ def _get_qc_path(self, config: Optional["TaskConfig"] = None):
         return paths
 
     def get_version(self) -> str:
+        # excuse missing Q-Chem for QCEngineRecords tests
+        found = self.found()
+        if not found:
+            return safe_version("0.0.0")
+
         self.found(raise_error=True)
```

Comment:
Perhaps remove this one since we have the above now?

Reply:
the `found()`s? second is normal behavior and has the `raise_error` behavior. first is the crazy escape path for qcer.

Reply:
But will the second ever have any effect? If `qchem` isn't found, the first will return the `safe_version("0.0.0")` call and exit the method. If `qchem` is found, it will sail through? I might be misunderstanding the subtleties of `qchem` here...

Reply:
yes, you're right. the return v0.0 was newly needed to appease qcer after I made qchem run for version. but the second with raise_error is the correct harness behavior. proper solution would be for the qcer tests to set an envvar that could be used to narrowly trigger the v0.0 special behavior. I'd like to keep the second `found` in memory of what we should return to.

I don't fully understand the subtleties of the qchem harness myself. it's one of those programs that settled into an idiosyncratic groove long ago, and it looks like DGAS had to add extra hoops to to stuff it into a harness.